src/Core/Support/ContactRepositoryInterface.php

```diff
@@ -38,5 +38,19 @@ interface ContactRepositoryInterface
      *
      * @return array
      */
-    public function findAllByLangId($langId);
+    public function findAllByLangId(int $langId);
+
+    /**
+     * Return available categories contacts.
+     *
+     * @return array<array<string, string|null>>
+     */
+    public function getCategoriesContacts(): array;
```

Comment:
You cannot add new methods to an existing interface as it is a breaking change You need to create a new one to include these two new methods (however a single service can implement both)

Reply:
You already added a new `PrestaShop\PrestaShop\Core\Domain\Contact\Repository\ContactRepositoryInterface` that's the one you need to use in your handlers The changes here should be reverted